src/auth/user-record.ts

```diff
@@ -326,7 +326,8 @@ export class UserMetadata {
     // These bugs have already been addressed since then.
     utils.addReadonlyGetter(this, 'creationTime', parseDate(response.createdAt));
     utils.addReadonlyGetter(this, 'lastSignInTime', parseDate(response.lastLoginAt));
-    utils.addReadonlyGetter(this, 'lastRefreshTime', parseDate(response.lastRefreshAt));
+    const lastRefreshAt = response.lastRefreshAt ? new Date(response.lastRefreshAt).toUTCString() : null;
+    utils.addReadonlyGetter(this, 'lastRefreshTime', lastRefreshAt);
```

Comment:
You also need to return this in `toJSON` and add a test for it
```
return {
  lastSignInTime: this.lastSignInTime,	
  creationTime: this.creationTime,
  lastRefreshTime: this.refreshTime,
};
```

Reply:
It's in the followup (#889)